app/services/ams/export/export_service.rb

```diff
@@ -47,6 +47,24 @@ def upload_to_s3
         end
         @s3_path = obj.public_url
       end
+
+      def scp_to_aapb(user)
+        # aapb_key_path = '-i /home/ec2-user/.ssh/aapb-zip-key.pem'
+        aapb_key_path = ""
+        filepath = @file_path.path
+        filename = File.basename(@file_path.path)
+        aapb_host = 'ec2-18-213-230-80.compute-1.amazonaws.com'
+        # config aapb host
+        if aapb_key_path && aapb_host.present? && filepath.present?
+
+          `scp #{aapb_key_path} #{filepath} ec2-user@#{aapb_host}:/home/ec2-user/ingest_zips/#{filename}`
+          output =  `ssh -t #{aapb_key_path} ec2-user@#{aapb_host} 'cd /home/ec2-user/ingest_zips && unzip #{filename} && cd /var/www/aapb/current && RAILS_ENV=production /usr/bin/ruby scripts/download_clean_ingest.rb --stdout-log --files /home/ec2-user/ingest_zips/*.xml'`
+          puts output
+          Sidekiq::Logging.logger.info output
```

Comment:
I think we'll probably just want `Rails.logger.info` here

Reply:
Also, we should provide some useful error messages if the `ENV['AAPB_HOST']` is not set, or if the AAPB machine is not reachable.